dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2774,3 +2774,70 @@ def test_divisions_with_null_partition(tmpdir, engine):
 
     ddf_read = dd.read_parquet(str(tmpdir), engine=engine, index="a")
     assert ddf_read.divisions == (None, None, None)
+
+
+def test_parquet_pyarrow_write_empty_metadata(tmpdir):
+    # https://github.com/dask/dask/issues/6600
+    tmpdir = str(tmpdir)
+
+    df_a = dask.delayed(pd.DataFrame.from_dict)(
+        {"x": [], "y": []}, dtype=("int", "int")
+    )
+    df_b = dask.delayed(pd.DataFrame.from_dict)(
+        {"x": [1, 1, 2, 2], "y": [1, 0, 1, 0]}, dtype=("int64", "int64")
+    )
+    df_c = dask.delayed(pd.DataFrame.from_dict)(
+        {"x": [1, 2, 1, 2], "y": [1, 0, 1, 0]}, dtype=("int64", "int64")
+    )
+
+    df = dd.from_delayed([df_a, df_b, df_c])
+
+    try:
+        df.to_parquet(
+            tmpdir,
+            engine="pyarrow",
+            partition_on=["x"],
+            append=False,
+            compute_kwargs={"scheduler": "synchronous"},
+        )
+
+    except AttributeError:
+        pytest.fail("Unexpected AttributeError")
+
+
+def test_parquet_pyarrow_write_empty_metadata_append(tmpdir):
+    # https://github.com/dask/dask/issues/6600
+    tmpdir = str(tmpdir)
+
+    df_a = dask.delayed(pd.DataFrame.from_dict)(
+        {"x": [1, 1, 2, 2], "y": [1, 0, 1, 0]}, dtype=("int64", "int64")
+    )
+    df_b = dask.delayed(pd.DataFrame.from_dict)(
+        {"x": [1, 2, 1, 2], "y": [2, 0, 2, 0]}, dtype=("int64", "int64")
+    )
+
+    df1 = dd.from_delayed([df_a, df_b])
+    df1.to_parquet(
+        tmpdir,
+        engine="pyarrow",
+        partition_on=["x"],
+        append=False,
+        compute_kwargs={"scheduler": "synchronous"},
```

Comment:
This is just for debugging convenience, correct? Can we remove for the test?

Reply:
Correct, removed and pushed. Thanks